docs/community/local-groups.mdx

```diff
@@ -173,3 +173,6 @@ us on [Discord](https://discord.com/invite/ktMAKGBnBs) to add your group.
 
 ### Wisconsin
 - [Meshconsin](https://meshconsin.org)
+
+### Japan
+- [Meshtastic Japan](https://www.facebook.com/share/g/BQCGxZhw9SxFQja8/?mibextid=K35XfP)
```

Comment:
Hello, could you please change the listing here to match your group, "Meshtastic Japan Community"

I would also request that you use the M-Powered Logo, or an original design, on the facebook group.

Thank you and I look forward to adding your group.

Reply:
I changed the name. Is it good?

Reply:
Does the logo need to be smaller?

Reply:
> I changed the name. Is it good?

The link here still says "Meshtastic Japan"

Reply:
> Does the logo need to be smaller?

It's not so much the size as it needs to be "secondary" to your own identity as a club, or you may wish to use the M-Powered Logo:
https://github.com/meshtastic/design/tree/master/Meshtastic%20Powered%20Logo

Reply:
https://github.com/meshtastic/meshtastic/compare/master...Goyath:patch-7?expand=1
Sorry, I'm not familiar with this, I may have rewritten it somewhere else. Please check.